index.html

```diff
@@ -27,13 +27,23 @@
       <img src="images/jupyter-sq-text.svg" alt="Jupyter" id="jupyter-logo"/>
     </div>
     <div class="row">
-    <div id="sub-title" class="col-md-8 col-md-offset-2">
-      <h2 class="text-center">Evolved from the IPython Project</h2>
-      <h2 class="text-center">
-        The language-agnostic parts of IPython are getting a new home in Project Jupyter
-      </h2>
+      <div id="sub-title" class="col-md-8 col-md-offset-2">
+        <h2 class="text-center">Evolved from the IPython Project</h2>
+        <h2 class="text-center">
+          The language-agnostic parts of IPython are getting a new home in Project Jupyter
+        </h2>
+      </div>
     </div>
+    <hr>
+    <div class="row">
+      <div class="col-md-8 col-md-offset-2 text-center">
+        <p>
+        <a type="button" class="btn btn-primary btn-lg" href='https://try.jupyter.org'>Try it In your browser</a>
+        </p>
+        <p class=''>No install or login required, with Python 3, Julia and R</p>
```

Comment:
What's the empty class for?

I'd make this text read: "No install or login required. Python 3, Julia, and R."